agreements/project-contribution/asset-transfer-agreement.md

```diff
@@ -0,0 +1,216 @@
+
+Assignor hereby grants Sponsor a limited power of attorney, coupled with an interest, to sign, on Assignor's behalf, any additional documents that Assignor may need in order to record the assignment of the Assets with government offices.
+Sponsor will only do so if Assignor does not sign the documents within a reasonable period of time after Sponsor's request.
+
+### Effective Date
+
+This Transfer Agreement is effective on the Effective Date above.
+
+### Governing Law
+
+This Transfer Agreement shall be governed by and construed in accordance with the laws of the State of Florida.
+
+### Controlling Agreement
+
+In the event of any conflict between the terms of this Transfer Agreement and the Fiscal Sponsorship Agreement, the terms of this Transfer Agreement shall control.
+
+### Amendments
+
+This Transfer Agreement may be amended only in writing signed by both parties.
+
+### Counterparts
+
+This Transfer Agreement may be executed in counterparts, each of which shall be deemed an original, but all of which together shall constitute one and the same instrument.
+
+By signing below, the parties agree to the terms and have executed this Asset Transfer Agreement as of the Effective Date.
+
+- **Assignor:** `__________________________________`  
+    `By:    ⎣_____________________________________________⎦`  
+    `Name:  ⎣_____________________________________________⎦`  
+    `Title: ⎣_____________________________________________⎦`  
+    `Date:  ⎣_____________________________________________⎦`  
+
+- **Sponsor:** *Commonhaus Foundation, Inc.*  
+    `By:    ⎣_____________________________________________⎦`  
+    `Name:  ⎣_____________________________________________⎦`  
+    `Title: ⎣_____________________________________________⎦`  
+    `Date:  ⎣_____________________________________________⎦`  
+
+## Schedules
+
+### SCHEDULE A: Registered Copyrights
+
+| Title | Country | Reg No |
+| ----- | ------- | ------ |
+| __    | __      | __     |
+
+&nbsp;
+
+### SCHEDULE B: Domain Names
+
+| Domain name | Registrar |
+| ----------- | --------- |
+| __          | __        |
+
+&nbsp;
+
+### SCHEDULE C: Hosting Accounts
+
+(List of web hosting accounts, document hosting platforms, build and testing servers, and cloud infrastructure)
+
+- __
+
+&nbsp;
+
+#### Source code repositories
+
+(List of individual repositories or organizations/groups of repositories)
+
+- __
+
+&nbsp;
+
+### SCHEDULE D: Social Media and Email Accounts
+
+(List of project-specific social media and email accounts)
+
+- __
+
+&nbsp;
+
+### SCHEDULE E: Trademarks
+
+#### Registered Trademarks
+
+| Trademark | Country | Appln Date | Appln No | Reg Date | Reg No |
+| --------- | ------- | ---------- | -------- | -------- | ------ |
+| __        | __      | __         | __       | __       | __     |
+
+&nbsp;
+
+#### Unregistered Trademarks
+
+(List; should be the same as what is in the Trademark List)
```

Comment:
Not sure what this means. You mean the list above, or do you mean the list we supplied in YAML when we applied?

Reply:
Yep. It was a hint to us that we already know the answer to this question.

When the agreement is signed, the marks being transferred should be in the Trademark List.